src/main/kotlin/blackjack/model/Hand.kt

```diff
@@ -0,0 +1,28 @@
+package blackjack.model
+
+class Hand(val cards: List<PlayingCard> = emptyList()) {
+    fun calculateHand(): Int {
+        val values = cards.map { it.value }
+        var aceCounts = values.count { it == BIG_ACE_VALUE }
+        var score = values.sum()
+        while (aceCounts > 0 && score > BLACKJACK_VALUE) {
+            score -= ACE_REDUCE_VALUE
+            aceCounts--
+        }
+        if (score < ACE_REDUCE_VALUE && values.contains(BIG_ACE_VALUE)) {
+            score += ACE_REDUCE_VALUE
+        }
+        return score
+    }
+
+    fun toText(): String {
```

Comment:
If it's something needed in the View layer, how about handling it in the View instead of the Domain layer? 🙂